Example/Example/MoreAttachmentRenderer.swift

```diff
@@ -30,7 +30,11 @@ extension MoreAttachmentRenderer: TextViewAttachmentImageProvider {
         UIGraphicsBeginImageContextWithOptions(size, false, 0)
 
         let label = attachment.text.uppercased()
-        let attributes = [NSForegroundColorAttributeName: textColor]
+
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.baseWritingDirection = .leftToRight
```

Comment:
I know auto-layout is a bit of a headache, but it certainly feels like we're missing something if we need to force this setting here.

That said, workarounds can be acceptable if we document them explaining why they're needed.  Maybe even adding a link to this PR or the discussion.

The reason being: the next developer working on this code may very well remove this line without knowing what's affected by it.